examples/dropwizard/config.yml

```diff
@@ -0,0 +1,16 @@
+hello:
```

Comment:
Is it possible to move this file to `src/main/jib`? Even if that requires some custom path configuration on the Dropwizard size, I think it will reduce the complexity originated mainly from it: no need for copy-rename-maven-plugin, maven-clean-plugin, the weird copying into the src directory in the generate-source phase while preventing committing the generated file into the repo, and no need to configure `extraDirectory`. And I think it will allow removing the `package` and `deploy` binding altogether. I just hope Dropwizard is flexible enough to place `config.yml` in a non-project root. The example feels a bit heavy on the config side overall as a getting started experience (perhaps the maven archetype is not intended to be a minimal example?), so it'd be nice if we can simplify it.

Reply:
It does seem odd to copy this `config.yaml` into the root of the container.

Reply:
If so, it sounds like we can move `config.yml` to `src/main/jib/app/config.yml` (or `dropwizard-config.yml`) and have `<args><arg>server</arg><arg>/app/config.yml</arg></args>`. Then we can remove copy-rename-maven-plugin, maven-clean-plugin, `config.yml` from `.gitignore`, and the `<extraContents>` property. Probably at this point, I think we can also remove the phase binding, since `mvn compile jib:build` or `mvn compile jib:dockerBuild` will work out of the box.

Reply:
That's a good point about just having the config be in `src/main/jib/app` rather than mucking with copying/cleaning it. 

Can do 👍